### PR TITLE
Add basic auth

### LINKED ISSUE #11: Add authentication
Add some authentication to protect the administrator pages from being viewed

### PR DESCRIPTION
**Pull request by zyhou**
closes #11 

- [x] add auth on all page
- [x] clean env var

Just add `apero_web_username` and `apero_web_password` in now secrets :)

### CODE REVIEW

**Comment by Euregan** [comment]
#### 📝Changed routes:


| [**/**](https://aperoweb-git-auth-basic.euregan.now.sh/) | [**/talks**](https://aperoweb-git-auth-basic.euregan.now.sh/talks) |
|:-:|:-:|
|<a href="https://aperoweb-git-auth-basic.euregan.now.sh/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=microlink.nyc3.digitaloceanspaces.com/FdyC3rlINHsRIWK1TzfOEyS9Gjg_">(view full size)</a> |<a href="https://aperoweb-git-auth-basic.euregan.now.sh/talks">Screenshot of /talks</a><br /><sup><a href="https://images.weserv.nl?url=microlink.nyc3.digitaloceanspaces.com/ScphJL_Gg28VMy_NbrVN60c8ctk_">(view full size)</a> |







Commit 89886a72d346cf64fb8cf64502634ecf28975d3a (https://aperoweb-git-auth-basic.euregan.now.sh).

**Comment by zyhou** [comment]
Just add `apero_web_username` and `apero_web_password` in now secrets :)

File: now.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
     },
     "build": {
         "env": {
-            "AIRTABLE_TOKEN": "@airtable_token"
```
Comment (Euregan) [inline now.json:7]: Why delete that?

Comment (zyhou) [reply now.json:7]: No need to put the token on the front side

**Review by Euregan — APPROVED** [review]

**Pull request merged**